app/types/crashesList.ts

```diff
@@ -1,4 +1,7 @@
-import { z } from "zod";
-import { crashesListSchema } from "@/schema/crashesList";
-
-export type CrashesListCrash = z.infer<typeof crashesListSchema>;
+export type CrashesListRow = {
```

Comment:
`CrashesListRow` seemed less awkward than `CrashesListRow` 🤷

Reply:
I like that a lot—thinking in terms of what the purpose of this view is rather than describing how the data is presented. I'll add a todo to reconsider this naming convention 🙏